modules/metering-install-configure-storage.adoc

```diff
@@ -0,0 +1,215 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adoc
+
+[id="metering-install-configure-storage_{context}"] 
+= Configuring Metering storage
+
+Configure the metering operator by editing the default metering configuration file created when you install the metering stack. Some configuration options, such as storage, must be configured before installing the metering stack. Use the documentation below and make changes to the `spec` section of the default configuration file.
+
+[NOTE]
+====
+The Configuring Metering documentation covers all available
+configuration options for the Metering Operator.
+====
+
+Metering requires persistent storage in two main ways. First, the data collected
+for reporting and the results of reports is usually kept in some form of object
+storage or distributed file system. You can configure metering to store this
+data locally or in a cloud provider of your choice.
+
+Second, a Hive metastore manages data to improve the efficency and speed of
+metering reports. By default, this information is stored in an embedded
+database called Derby, which keeps it's data on disk in a PersistentVolume. For
+larger deployments, this metadata can also be configured to use an existing MySQL or PostgreSQL database,
+instead of Derby. Read the documentation on configuring the Hive metastore for
+more details.
+
+[id="metering-storing-data-in-amazon-s3_{context}"]
+== Storing data in Amazon S3
+
+To use Amazon S3 for storage, you must must specify an S3 bucket, region, and
+secretName in the `spec` section your metering configuration file.
+
+The bucket and region values should be the name of the bucket and region,
+respectively. You can optionally include the path within the bucket where you wish to store Metering data.
+
+If you want to provide an existing S3 bucket, or do not want to provide IAM
+credentials that have CreateBucket permissions, set `createBucket` to `false` and provide the name of a
+pre-existing bucket.
+
+Below is an example configuration file:
+
+[source,yaml]
+----
+apiVersion: metering.openshift.io/v1
+kind: MeteringConfig
+metadata:
+  name: "operator-metering"
+spec:
+  storage:
+    type: "hive"
+    hive:
+      type: "s3"
+      s3:
+        bucket: "bucketname/path/"
+        region: "us-west-1"
+        secretName: "my-aws-secret"
+        # Set to false if you want to provide an existing bucket, instead of
+        # having Metering create the bucket on your behalf.
+        createBucket: true
+----
+
+The secretName should be the name of a secret in the Metering namespace containing the AWS credentials in the data.aws-access-key-id and data.aws-secret-access-key fields.
```

Comment:
Since they're literal fields, I would put backticks around secretName, data.aws-access-key-id and data.aws-secret-access-key.

Same elsewhere in the PR.